package.json

```diff
@@ -1,16 +1,23 @@
 {
-  "name": "express-authorshaven",
+  "name": "express-barefoot-nomad",
   "version": "1.0.0",
-  "description": "A Social platform for the creative at heart",
+  "description": "Making company travel and accomodation easy and convinient",
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "nodemon --exec node -r dotenv/config ./src/index.js",
-    "build": "node -r dotenv/config ./src/index.js"
+    "production": "npm run build && node dist/index.js",
```

Comment:
By convention the `npm start` script should be the `production` script. Also, since `nodemon` is used solely for development can we rename the script to `watch`?

Reply:
@timolinn so I should also change also the `Procfile` to point the script npm start??